profiles/arch/arm64/package.use.mask

```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Felix Janda <felix.janda@posteo.de> (2020-08-01)
+# Requires unkeyworded dev-lang/spidermonkey:17
+>=www-client/elinks-0.13.3 javascript
```

Comment:
If we end up sticking with :17, remember to request keywording or at least investigate if :17 (which is old?) has support for these arches.